wordpress-mobile/WordPress-iOS
# [Jetpack Content Migration] Update background color in Dark Mode to match the design

## Related PRs
The following PRs should be merged before this one:
- https://github.com/wordpress-mobile/WordPress-iOS/pull/19632
- https://github.com/wordpress-mobile/WordPress-iOS/pull/19661

## Description
This PR changes the background color of all migration screens from "Black" to "Jetpack Green Shade 100" to match the Figma design.

Figma reference qk5s4q4mZsXknwhMWo2vIY-fi-5997%3A18179&t=QuRcyiCwlvk8zhpL-4

| Before | Notifications Step |
| ------ | --------------- |
| ![before](https://user-images.githubusercontent.com/9609223/203835023-dce1c605-ce4c-4f9b-8464-0db2a66b8f8b.png) | ![](https://user-images.githubusercontent.com/9609223/203833001-2ee1f2ac-3a76-467b-9c1e-7e8a9f9a1aab.png) |

## Test Instructions

<details><summary>Prerequisites</summary>
<p>

1. Open Xcode and go to `MigrationFlowCoordinator.swift`
2. Force `shouldSkipNotificationsScreen()` to return `false`. This will force the Notifications screen to appear even notifications permission is granted.
2. Build and run Jetpack app then log into your account
3. My Site tab > "Me" button >  App Settings > Debug
4. Enable "Content Migration" Feature Flag
 
</p>
</details>

1. Close and re-open the Jetpack app
2. Turn on Dark Mode
3. **Verify** the background color of all Migration screens matches the Figma design and screenshots

## Regression Notes
1. Potential unintended areas of impact
None.

6. What I did to test those areas of impact (or what existing automated tests I relied on)
None.

7. What automated tests I added (or what prevented me from doing so)
None.

## PR submission checklist:
